docs/docs-content/getting-started/aws/deploy-manage-k8s-cluster-tf.md

```diff
@@ -0,0 +1,748 @@
+---
+sidebar_label: "Cluster Management with Terraform"
+title: "Cluster Management with Terraform"
+description: "Learn how to deploy and update a Palette host cluster to AWS with Terraform."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 50
+toc_max_heading_level: 2
+tags: ["getting-started", "aws", "terraform"]
+---
+
+The [Spectro Cloud Terraform](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs) provider
+allows you to create and manage Palette resources using Infrastructure as Code (IaC). With IaC, you can automate the
+provisioning of resources, collaborate in infrastructure changes, and maintain a single source of truth for your
+infrastructure.
+
+This tutorial will teach you how to use Terraform to deploy and update an Amazon Web Services (AWS) host cluster. You
+will learn how to create two versions of a cluster profile with different demo applications, update the deployed cluster
+with the new profile version, and then perform a cluster profile rollback.
+
+## Prerequisites
+
+To complete this tutorial, you will need the following items in place:
+
+- Tenant admin access to Palette.
+- Follow the steps described in the [Set up Palette with AWS](./setup.md) guide to authenticate Palette for use with
+  your AWS cloud account and create a Palette API key.
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/) or [Podman](https://podman.io/docs/installation)
+  installed if you choose to follow along using the tutorial container.
+- If you choose to clone the repository instead of using the tutorial container, make sure you have the following
+  software installed:
+  - [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) v1.9.0 or greater
+  - [Git](https://git-scm.com/downloads)
+  - [Kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+## Set Up Local Environment
+
+You can clone the [Tutorials](https://github.com/spectrocloud/tutorials) repository locally or follow along by
+downloading a container image that includes the tutorial code and all dependencies.
+
+<Tabs>
+
+<TabItem label="Docker" value="docker">
+
+Start Docker Desktop and ensure that the Docker daemon is available by issuing a command to list the currently active
+containers.
+
+```bash
+docker ps
+```
+
+Use the following command to download the tutorial image. This image includes the required software and Terraform files.
```

Comment:
Can we remove this docker pull and eliminate this step since Docker will download the image anyways?

Reply:
Yes :)